contracts/evm/library/utils/Types.sol

```diff
@@ -71,6 +70,11 @@ library Types {
         bytes data;
     }
 
+    struct PersistentMessage {
+        int messageType;
```

Comment:
Type should not be here like CallMessage

Reply:
So they are basically only name abstractions, struct might not even be needed in a sense